bin/iron/build.rs

```diff
@@ -1,3 +1,6 @@
 fn main() {
+    println!("cargo:rerun-if-changed=bin/iron/token_list.json");
+    iron_token_list::build();
```

Comment:
this one was suposed to be the build script of the token_list crate itself, not called from here